pkg/collector/corechecks/cluster/orchestrator/collectors/ecs/task_test.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build orchestrator
+
+package ecs
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestInitClusterID(t *testing.T) {
```

Comment:
Would it be possible to add a test for the other task functions?

Reply:
This collector is only used by the new check so they are tested in this test
https://github.com/DataDog/datadog-agent/blob/0a9bc127fc80ffa5ad4d240361a8e3bbf8aa26c4/pkg/collector/corechecks/orchestrator/ecs/ecs_test.go#L74